src/svgcontext.ts

```diff
@@ -104,10 +100,9 @@ export class SVGContext extends RenderContext {
 
     this.path = '';
     this.pen = { x: NaN, y: NaN };
-    this.lineWidth = 1.0;
 
     const defaultFontAttributes = {
-      'font-family': Metrics.get('fontFamily'),
+      'font-family': Metrics.get('fontFamily') as string,
```

Comment:
Curious why this cast was needed? Is it not normally a string?

Reply:
get returns any :(